vscode/src/webview/webview.tsx

```diff
@@ -178,6 +246,12 @@ function App({ state }: { state: State }) {
       return <CircuitPanel {...state.props}></CircuitPanel>;
     case "help":
       return <HelpPage />;
+    case "documentation":
+      // Ideally we'd have this on all web views, but it makes the font a little
```

Comment:
Per the comments, only added this class to the docs now, not the webview in general, as I don't want to impact rendering of other existing views.